src/api_server/src/request/machine_configuration.rs

```diff
@@ -91,13 +100,13 @@ mod tests {
         // 2. Test case for mandatory fields.
         let body = r#"{
                 "mem_size_mib": 1024,
-                "ht_enabled": true
+                "ht_enabled": false
```

Comment:
Not clear why this were changed from true to false?

Reply:
So that those unit tests don't fail on ARM